packages/react/src/lib/mergeThemes.ts

```diff
@@ -212,6 +216,8 @@ const mergeThemes = (...themes: ThemeInput[]): ThemePrepared => {
 
     acc.componentVariables = mergeThemeVariables(acc.componentVariables, next.componentVariables)
 
+    acc.componentVariants = mergeComponentVariants(acc.componentVariants, next.componentVariants)
```

Comment:
policy of merging variants' sets; in case of name collisions (`Button` -> `EmojiButton` in source theme, `Button`-> `EmojiButton` in theme to merge) the last one will win